### PR TITLE
open-cluster-management/backlog#5117 -- new topic on console

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: manage_applications/app_console.adoc [hunk 1]
```diff
@@ -1,142 +1,107 @@
...
-* Access a topology view that encompasses the new application resource definitions, including channels, subscriptions, and placement rules.
+* Visualize deployed applications across your clusters, including any associated channels and subscriptions.
+
+* Access a topology view that encompasses the new application resource definitions, including channels, subscriptions, placement rules, and Ansible Tower jobs.
+
 * View individual status in the context of an application, including deployments, updates, and subscriptions.
-* Add and edit channels, subscriptions, placement rules, and applications.
 
-Samples for all resources are located in the xref:../manage_applications/app_sample.adoc#application-samples[Application resource samples] documentation.
+* Add and edit channels, subscriptions, placement rules, and applications.
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: We don't have support in 2.1 to create channel, subs, PR by themself, they all get created under the cover when you create an application using the new dialog.
You can only view, edit channel, subs, PR ( UI to be done in sprint 4 ) . The pipeline is replaced by tables , with actions to view channel, subs, PR 

I would rephrase : Create and edit applications. Use the _Advanced configuration_ to view or edit channel, subs, PR , if you have the right access.

File: manage_applications/app_console.adoc [hunk 2]
```diff
@@ -1,142 +1,107 @@
...
+The console includes a dashboard for managing the application lifecycle. You can use the console dashboard to create and manage applications resources, and view the status of your applications. Enhanced capabilities help your developers and operations personnel create, deploy, update, manage, and visualize applications across your clusters.
 
 See the following application console capabilities:
 
-* Use the topology view to visualize deployed applications across your clusters, including any associated channels and subscriptions.
-* Access a topology view that encompasses the new application resource definitions, including channels, subscriptions, and placement rules.
+* Visualize deployed applications across your clusters, including any associated channels and subscriptions.
+
+* Access a topology view that encompasses the new application resource definitions, including channels, subscriptions, placement rules, and Ansible Tower jobs.
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: I am not sure if we still want to talk about channels in the topology context. 
I would refer to Channel on when discussing the _Advanced configuration_ tab ( here also goes the notion on Subs, PR )

Proposed change:
Access a topology view that encompasses the new application resource definitions, including resource repositories, subscriptions, placement rules, and deployed resources including any optional pre and post deployment hooks using Ansible Tower tasks.

File: manage_applications/app_console.adoc [hunk 3]
```diff
@@ -1,142 +1,107 @@
...
 
-The console includes a dashboard for managing the application lifecycle. You can use the console dashboard to create and manage applications resources. You can also view the status of your applications. The dashboard includes enhanced capabilities, which your developers and operations personnel can use to create, deploy, update, manage, and visualize applications across your clusters.
+The console includes a dashboard for managing the application lifecycle. You can use the console dashboard to create and manage applications resources, and view the status of your applications. Enhanced capabilities help your developers and operations personnel create, deploy, update, manage, and visualize applications across your clusters.
 
 See the following application console capabilities:
 
-* Use the topology view to visualize deployed applications across your clusters, including any associated channels and subscriptions.
-* Access a topology view that encompasses the new application resource definitions, including channels, subscriptions, and placement rules.
+* Visualize deployed applications across your clusters, including any associated channels and subscriptions.
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: Change : ( I would not refer to channels in the single app topo context, just Advanced configuration tab )

Visualize deployed applications across your clusters, including any associated resource repositories subscriptions and placement configurations.

File: manage_applications/app_console.adoc [hunk 4]
```diff
@@ -1,142 +1,107 @@
...
- ** The total number of channels and placement rules for all applications.
+From the main _Overview_ tab, see the following:
 
-You can click each summary card to open the _Search_ page and display more information about the selected resource.
 
-* From this page, you can also click the multiple *Create* options to create more resources with YAML editors. 
+* A table that lists all applications
+* The _Find resources_ box to filter the applications that are listed
+* The application name and namespace
+* The remote and local clusters where resources are deployed through a subscription
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: Then _number of_ remote and local clusters where resources are deployed through a subscription

File: manage_applications/app_console.adoc [hunk 5]
```diff
@@ -1,142 +1,107 @@
...
+* The date when the application was created
```
Comment (birsanv) [inline manage_applications/app_console.adoc:35]: Date is the last column

File: manage_applications/app_console.adoc [hunk 6]
```diff
@@ -1,142 +1,107 @@
...
+* A Time window constraint, if any were added
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: You are missing the Repositories column: The list of repositories where the definitions for the resource deployed by the application where located.

File: manage_applications/app_console.adoc [hunk 7]
```diff
@@ -1,142 +1,107 @@
...
+* More actions, such as *Delete application*.
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: actions : Edit, View Search app 
Actions available ( active ) if the user can execute them

File: manage_applications/app_console.adoc [hunk 8]
```diff
@@ -1,142 +1,107 @@
...
+* Resource topology
 
- ** Each row provides details for a single application. Rows are included for all of your applications.
-If needed, you can use the search box to filter the application rows to find a matching application.
- ** Expand the row for each application to view more details for the associated channels, subscriptions, and placement rules, view related incidents, and view the status of resource (pod) deployments.
- ** Select an associated subscription for a channel and application to view more details about that subscription, including the subscribed deployables, the associated channel, namespace, and host-cluster, the status of any related resource (pod) deployments, and the placement settings that are used with the subscription.
+[#resource-topology]
+== Resource topology
 
+The topology provides a visual representation of all the applications and resources within a project, their build status, and the components and services associated with them.
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: The topology provides a visual representation of the application that was selected including the resources deployed by this application on target clusters.

File: manage_applications/app_console.adoc [hunk 9]
```diff
@@ -1,142 +1,107 @@
...
 
-[#application-dashboard-single-applications]
-=== Application dashboard (single applications)
+* You can select any component from the topology view to view more details.
 
-From the _Overview_ tab of the dashboard for a single application, you can complete the following tasks:
+* View the details for a pod by clicking on the nodes to open the properties view.
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: Remove this, pod is no longer showing

Comment (birsanv) [reply manage_applications/app_console.adoc]: In general : View the deployment details for any resource deployed by this application by clicking on the resource node to open the properties view

File: manage_applications/app_console.adoc [hunk 10]
```diff
@@ -1,142 +1,107 @@
...
- ** The total number of managed clusters where the application is used.
-This card also shows the total number of subscriptions for the application on those managed clusters, and the number of subscriptions that failed to deploy to those managed clusters.
- ** The number of pods for the application, including the number that are running and failed.
- ** The number of policy violations and incidents that are related to the application.
-You can click each summary card to open the _Search_ page and display more information about the selected resource.
-* Select to view additional details for the application, including the application namespace, version, creation date, associated labels and annotations, and more.
-* View the resource topology for the application that shows the application and related components such as channels, deployables, related services, pods, and placement rules.
-You can select to edit the topology and update the YAML definitions for the application and related resources, such as to add, change, or remove resources.
+* View cluster CPU and memory.
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: From the cluster node , on the properties dialog view the CPU ..

File: manage_applications/app_console.adoc [hunk 11]
```diff
@@ -1,142 +1,107 @@
...
-* Select to view additional details for the application, including the application namespace, version, creation date, associated labels and annotations, and more.
-* View the resource topology for the application that shows the application and related components such as channels, deployables, related services, pods, and placement rules.
-You can select to edit the topology and update the YAML definitions for the application and related resources, such as to add, change, or remove resources.
+* View cluster CPU and memory. 
++
+**Note:** The cluster CPU and memory percentage that is displayed is the percentage that is currently utilized. This value is rounded down, so a very small value might display as `0`.
++
 
-From the _Resources_ tab for a single application, you can complete the following tasks:
+*  View a successful Ansible Tower deployment. Additionally, you can see errors if your Ansible Tower deployment is not successful.
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: If using Ansible tasks as prehook or posthook for the deployed application, you see these tasks in the application topology.
Click on the .. and you can see the details about the Ansible task deployment, including Ansible Tower Job URL and template name.

File: manage_applications/app_console.adoc [hunk 12]
```diff
@@ -1,142 +1,107 @@
...
 
-* View the resource list for the application.
-This list includes all resources that are associated with the application, including pods, secrets, services, and more.
-This list indicates the name, namespace, kind, API groups, and status of each resource.
-The list also indicates the cluster where the resource is deployed, the creation date of the resource, and the last date that the resource was updated.
-* View the resource pipeline for the application, which includes the summary of the application resources, options to create resources, and a table that provides more details about the resources.
-The resource pipeline for a single application includes similar information to the pipeline for all applications.
-The difference between the two pipelines is that this pipeline is scoped to just the selected application.
+* Click *Launch resource in Search* to search for related clusters.
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: Click *Launch resource in Search* to search for related resources.

File: manage_applications/app_console.adoc [hunk 13]
```diff
@@ -1,142 +1,107 @@
...
+| `Application`
+
 |===
 
 You can also search by other fields, including name, namespace, cluster, label, and more.
 
 From the search results, you can view identifying details for each resource, including the name, namespace, cluster, labels, and creation date.
 
-If needed, you can also expand the _Options_ menu in the search results for a resource to select to delete that resource.
+If needed, you can also expand *Actions* menu in the search results for a resource to select to delete that resource.
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: Hm.. if the delete action is available, and the user has right access .. this should go under search doc

File: manage_applications/app_console.adoc [hunk 14]
```diff
@@ -1,142 +1,107 @@
...
 
-By clicking the resource name in the search results, a YAML editor opens and displays the YAML definition for the resource.
-You can choose to edit the definition within the editor.
-Any changes that you save are applied to the resource immediately.
+Click the resource name in the search results to open the YAML editor. Any changes that you save are applied to the resource immediately.
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: Not sure if we want to document this here , this is Search territory.

We would document that 
- you can click on _View Resource YAML_ from the topology property dialog on a resource and this takes you to the Search editor.
- you can click on _Launch resource in Search_ from the topology property dialog on a resource and takes you the Search page

File: manage_applications/app_console.adoc [hunk 15]
```diff
@@ -1,142 +1,107 @@
...
 
-[#resource-topology]
-== Resource topology
 
-The application topology includes a visualization of application cards that display status. The topology view for each application includes any services, deployments, charts, and pods for that application.
+[#Advanced-configuration]
+== Advanced configuration
 
-* You can select any component from the topology view to view more details.
+Click the *Advanced configuration* to vbiew the summary of resources for all applications on different cards. These resource summary cards show the following information about your applications and related managed clusters:
```
Comment (birsanv) [inline manage_applications/app_console.adoc]: Remove this, cards no longer show under Advanced configuration

Comment (birsanv) [reply manage_applications/app_console.adoc]: This tab is changed completely, it shows Channel, Sub, PR tables

https://marvelapp.com/prototype/24ib00hg/screen/71747880

**Review by birsanv — CHANGES_REQUESTED** [review]

File: manage_applications/app_console.adoc (hunk 13 above)
Comment (swopebe) [reply manage_applications/app_console.adoc]: it's in the search area and I changed it a bit

**Review by birsanv — APPROVED** [review]
